src/lib/cache/URLDataCache.js

```diff
@@ -1,55 +1,58 @@
 // @flow
 import { globalFetchManager } from '../FetchManager';
 import { t } from 'ttag';
+
 import ResponseError from '../ResponseError';
+import TTLCache, { type TTLCacheOptions } from './TTLCache';
 
 // Provides a WhatWG-fetch-like API to make HTTP requests.
 // Caches response promises and returns an old promise if one is existing for the same URL.
 
+type URL = string;
```

Comment:
-1: I think this is misleading, as URL looks like a complex type on first sight, and there is a `URL` class on window in the browser. I'd leave `string` as key type – it's clear what it does and it does not imply the thought that there might be some canonical URL recognition tech behind it. Also, right in the subclass it's no URL anymore, but a hostname.